readme.md

```diff
@@ -21,21 +21,71 @@ Previously the tests were maintained as part of the system operations. In order
 to allow other developers to work on maintaining the tests without being
 committers to the Ops project, these tests now standalone.
 
-## Setup
+## Running the tests
+
+There are two ways to run the tests, either in a docker container or natively.
+
+The docker container provides a stable software environment for running the tests.
+Running natively does not require docker to be installed.
+
+##  Running with Docker
+
+### Setup
+
+- Docker must be installed and the docker daemon running
+- the aws credentials helper should be installed and configured - 
+  see https://github.com/awslabs/amazon-ecr-credential-helper/blob/master/README.md
+  The AWS_PROFILE should be set - defaults to 'lr'
```

Comment:
This needs more explanation. I can see that `AWS_PROFILE=lr` need to be in the environment, but I'm not sure what you're expecting to be in `~/.aws/credentials` to make that meaningful. I don't have an `lr` section in my AWS creds at the moment

Reply:
I have attempted to clarify.